packages/analytics/amplify_analytics_pinpoint_dart/test/queued_item_store_test.dart

```diff
@@ -0,0 +1,299 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import 'package:amplify_analytics_pinpoint_dart/src/impl/analytics_client/event_client/queued_item_store/dart_queued_item_store.dart';
+
+import 'package:test/test.dart';
+
+void main() {
+  late DartQueuedItemStore db;
+
+  group('DartQueuedItemStore ', () {
+    setUpAll(() {
+      db = DartQueuedItemStore('test');
```

Comment:
*nit:* This creates a file in the current working dir, which is not cleaned up. In Amplify DB Common, I had used `/tmp` as the path. A better solution might be to use a temp dir and also clean it up in a tearDownAll step. I would at least use `/tmp`.

Reply:
switched to /tmp